e2e/support/helpers/api/updateSetting.ts

```diff
@@ -7,5 +7,10 @@ export const updateSetting = <
   setting: TKey,
   value: TValue,
 ): Cypress.Chainable<Cypress.Response<never>> => {
+  // TODO: remove before merging integration branch
+  if (setting === "enable-embedding-static") {
+    updateSetting("enable-embedding", value as boolean);
```

Comment:
Is this to make embedding work with the current backend?

Reply:
exactly, without these small changes we'd have all tests fail and we won't be able to see the real failures among the ones caused by the FE already being on the new settings while the BE is not

Reply:
I see, it all makes sense now. Thanks!